### PR TITLE
Fix #188: Update Docker dependencies

### LINKED ISSUE #188: Update Docker dependencies

### PR DESCRIPTION
**Pull request by banterCZ**
- ibm-semeru 17.0.9_9-jre

### CODE REVIEW

File: deploy/dockerfile/database/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+FROM ibm-semeru-runtimes:open-17.0.9_9-jre
 
 LABEL maintainer="Petr Dvořák <petr@wultra.com>"
 
 # Prepare environment variables
 # LIQUIBASE_ prefix must not be used, because this is only supported in PRO version
 ENV JAVA_HOME=/opt/java/openjdk \
     LB_HOME=/usr/local/liquibase \
-    LB_VERSION=4.23.2 \
+    LB_VERSION=4.25.1 \
```
Comment (zcgandcomp) [inline deploy/dockerfile/database/Dockerfile:10]: Do not use LB 25.1. as it implies of use of Oracle 23c
see [Release Notes  for 24.0](https://docs.liquibase.com/start/release-notes/liquibase-release-notes/liquibase-4.24.0.html) 
"Liquibase now supports Oracle's new BOOLEAN data type that was released in Oracle 23c. Prior to this update, Liquibase converted the BOOLEAN data type to the type NUMBER(1). [[#4579](https://github.com/liquibase/liquibase/pull/4579)]"

Or better put - does LB support old Oracle and converts the boolean to NUMBER(1) ?
How does this behave when generating SQL using updateSQL command?

Till that time let's stay on LB `4.23.2`

And unless there are some security vulnerabilities to fix I prefer to update to new version of any dependencies in the beginning of release cycle.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
JRE update fine
LB update - please postpone

File: deploy/dockerfile/database/Dockerfile (hunk 1 above)
Comment (banterCZ) [reply deploy/dockerfile/database/Dockerfile:10]: But the release notes also contain
> To improve compatibility, the Oracle JDBC driver was downgraded to the most recent long-term support (LTS) version, series 19.x. The previously shipped Oracle JDBC driver was a development version (version 23.x).

Comment (banterCZ) [reply deploy/dockerfile/database/Dockerfile]: Liquibase version reverted.

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**